### ISSUE TITLE
Unmet peer dependency

### ISSUE THREAD

**Issue by ffxsam** [davidkpiano/react-redux-form#215]
For some reason, `react-addons-shallow-compare` is not getting installed when I install `react-redux-form`.

```
Unable to resolve some modules:

  "react-addons-shallow-compare" in
/Volumes/SuperData/Sites/reelcrafter/node_modules/react-redux-form/lib/components/field-component.js
(web.browser)

If you notice problems related to these missing modules, consider running:

  meteor npm install --save react-addons-shallow-compare
```

**Comment by davidkpiano** [comment]
Yeah I'm thinking that dependency isn't common enough to warrant a peer-dependency. cc. @chrisblossom 

I'll add it as a dependency in the next patch.

**Comment by chrisblossom** [comment]
Peer dependencies aren't there because they are common. The reason ```react-addons-shallow-compare``` is a peer dependency is to preserve React 0.14 support. I don't think adding ```react-addons-shallow-compare``` warrants dropping support for 0.14.

On a side note, especially with pre-1.0 packages, you should pin your packages to specific versions and consider using npm shrinkwrap.

**Comment by davidkpiano** [comment]
@chrisblossom What are your thoughts on removing React 0.14 support?

**Comment by chrisblossom** [comment]
I can't speak for everyone, but I personally am still on 0.14 mainly because of [aickin/react-dom-stream](https://github.com/aickin/react-dom-stream), which doesn't support 15 because they are trying to get the changes into React core.

Think about when React 16 is released. If you are going to pin React down to a specific version, how long are you going to support v15? Or when would you start supporting v16?

**Comment by davidkpiano** [comment]
@ffxsam @chrisblossom Problem solved, I'm now doing this:

```diff
- import shallowCompare from 'react-addons-shallow-compare';
+ import shallowCompare from 'react/lib/shallowCompare';
```

which will grab the internal module from whichever version of React that is being used. That way, it is not required as an additional peer dependency, which would be redundant anyway, as it is already a part of the React module.

**Comment by ffxsam** [comment]
Awesome, thanks!

**Comment by davidkpiano** [comment]
Released as `0.12.6`!
